exercises/bank-account/BankAccountTest.fs

```diff
@@ -12,37 +12,65 @@ let ``Returns empty balance after opening`` () =
 [<Test>]
 [<Ignore("Remove to run test")>]
 let ``Check basic balance`` () =
-    let account1 = mkBankAccount() |> openAccount
-    let openingBalance = account1 |> getBalance 
+    let account = mkBankAccount() |> openAccount
+    let openingBalance = account |> getBalance 
 
-    let account2 = account1 |> updateBalance 10.0
-    let updatedBalance = account2 |> getBalance
+    let updatedBalance = 
+        account
+        |> updateBalance 10.0 
+        |> getBalance
 
     Assert.That(openingBalance, Is.EqualTo(Some 0.0))
     Assert.That(updatedBalance, Is.EqualTo(Some 10.0))
 
 [<Test>]
 [<Ignore("Remove to run test")>]
 let ``Balance can increment or decrement`` () =    
-    let account1 = mkBankAccount() |> openAccount
-    let openingBalance = account1 |> getBalance 
+    let account = mkBankAccount() |> openAccount
+    let openingBalance = account |> getBalance 
 
-    let account2 = account1 |> updateBalance 10.0
-    let addedBalance = account2 |> getBalance
+    let addedBalance = 
+        account 
+        |> updateBalance 10.0
+        |> getBalance
 
-    let account3 = account2 |> updateBalance -15.0
-    let subtractedBalance = account3 |> getBalance
+    let subtractedBalance = 
+        account 
+        |> updateBalance -15.0
+        |> getBalance
 
     Assert.That(openingBalance, Is.EqualTo(Some 0.0))
     Assert.That(addedBalance, Is.EqualTo(Some 10.0))
     Assert.That(subtractedBalance, Is.EqualTo(Some -5.0))
 
 [<Test>]
-[<Ignore("Remove to run test")>]
-let ``Account can be closed`` () =
+[<Ignore("Remove to run test")>]let ``Account can be closed`` () =
     let account = 
         mkBankAccount()
         |> openAccount
         |> closeAccount
 
-    Assert.That(account |> getBalance, Is.EqualTo(None))
\ No newline at end of file
+    Assert.That(account |> getBalance, Is.EqualTo(None))
+    
+[<Test>]
+[<Ignore("Remove to run test")>]
+let ``Account can be updated from multiple threads`` () =
+
```

Comment:
Super minor, but this extra newline is not consistent with the formatting in the rest of the tests